pubspec.yaml

```diff
@@ -42,30 +43,14 @@ flutter:
   # included with your application, so that you can use the icons in
   # the material Icons class.
   uses-material-design: true
-  # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
-  #   - images/a_dot_ham.jpeg
-  # An image asset can refer to one or more resolution-specific "variants", see
-  # https://flutter.dev/assets-and-images/#resolution-aware.
-  # For details regarding adding assets from package dependencies, see
-  # https://flutter.dev/assets-and-images/#from-packages
-  # To add custom fonts to your application, add a fonts section here,
-  # in this "flutter" section. Each entry in this list should have a
-  # "family" key with the font family name, and a "fonts" key with a
-  # list giving the asset and other descriptors for the font. For
-  # example:
-  # fonts:
-  #   - family: Schyler
-  #     fonts:
-  #       - asset: fonts/Schyler-Regular.ttf
-  #       - asset: fonts/Schyler-Italic.ttf
-  #         style: italic
-  #   - family: Trajan Pro
-  #     fonts:
-  #       - asset: fonts/TrajanPro.ttf
-  #       - asset: fonts/TrajanPro_Bold.ttf
-  #         weight: 700
-  #
-  # For details regarding fonts from package dependencies,
-  # see https://flutter.dev/custom-fonts/#from-packages
+  assets:
+    - assets/
+    - assets/images/
+    - assets/videos/
+    - assets/svg/
+    - assets/fonts/
+
+  fonts:
+    - family: neuzeitsltstd
+      fonts:
+        - asset: assets/fonts/neuzeit.otf
```

Comment:
we should add support for color also?
```
flutter_gen:
  colors:
    inputs:
      - assets/color/colors.xml
```
```
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <color name="white_sample">#FFFFFF</color>
</resources>
```

Reply:
@luongvo For my opinion, making `CustomColor` class inside the app is a good way for development and maintenance.
We can use the available color from `Material` as well.

Reply:
@doannimble but most of the time we receive color resources from design as hex code. It's better to keep those definitions as hex code in our XML config. Of course, we still able to use color from other resources such as Material, etc. How do you think?

Reply:
@luongvo We can define the custom color with hex code as well :
`static const Color greySolid01 = Color(0xFFB8B9C0);`

and AS also support the real vision for your reference:

<img width="545" alt="Screen Shot 2021-09-20 at 4 33 50 PM" src="https://user-images.githubusercontent.com/88996251/133981936-ad4c5d16-941e-41bc-b6ff-dbe94f7802cb.png">

Reply:
okay

Reply:
I think it's nice to have and both are valid; it depends on the project lead and the style we implemented from the beginning, how about making a vote and doing a different PR/Issue? 




https://user-images.githubusercontent.com/13435717/133982941-925c2908-699e-4dda-88da-00f718f3e508.mov

Reply:
@sleepylee I'm OK with making the vote 😄